src/parametric.cpp

```diff
@@ -963,7 +1209,7 @@ wxArrayString ParametricViewer::getFromExcel(const wxString& input_name, int& ro
 #endif // __WXMSW__
 }
 
-void ParametricViewer::SaveToCSV()
+void ParametricViewer::SaveToCSVOld()
```

Comment:
It looks like this isn't used anywhere now. I think my first choice would be to remove it, but if you feel strongly about keeping it, can you add a comment explaining the differences with the current function.